src/replica/ConfigWorker.cc

```diff
@@ -85,18 +106,23 @@ json WorkerInfo::toJson() const {
     infoJson["name"] = name;
     infoJson["is-enabled"] = isEnabled ? 1 : 0;
     infoJson["is-read-only"] = isReadOnly ? 1 : 0;
-    infoJson["svc-host"] = svcHost;
+    infoJson["svc-host"] = svcHost.addr;
+    infoJson["svc-host-name"] = svcHost.name;
```

Comment:
Why not 

    infoJson["svc-host"] = svcHost.toJson();

?

Reply:
Good suggestion! Will do so.